### PR TITLE
chore: add labels to docker image

### PR DESCRIPTION
**Pull request by garethjevans**

### CODE REVIEW

**Review by olblak — APPROVED** [review]

**Comment by garethjevans** [comment]
I don't have permission to merge this one @olblak

**Review by dduportal — APPROVED** [review]
LGTM 👍

**Comment by dduportal** [comment]
Neither do I :(

**Comment by olblak** [comment]
I just gave write permission to the group jenkins-infra/docker so next time you should be able to merge it @dduportal @garethjevans

**Review by olblak — CHANGES_REQUESTED** [review]

File: Makefile [hunk 1]
```diff
@@ -1,19 +1,42 @@
 .PHONY: build run shell mock
 
-COMMIT=$(shell git rev-parse HEAD | cut -c1-6)
+COMMIT         := $(shell git rev-parse HEAD | cut -c1-6)
+BUILD_DATE     := $(shell date +%Y-%m-%dT%H:%M:%SZ)
+GIT_TREE_STATE := $(shell test -z "`git status --porcelain`" && echo "clean" || echo "dirty")
+GIT_SCM_URL    := $(shell git config --get remote.origin.url)
+SCM_URI        := $(subst git@github.com:,https://github.com/,$(GIT_SCM_URL))
+BUILD_DATE     := $(shell date --utc '+%Y-%m-%dT%H:%M:%S' 2>/dev/null || gdate --utc '+%Y-%m-%dT%H:%M:%S')
```
[the remark this answers is not in the document]
Comment (garethjevans) [reply Makefile:8]: good point, i'll fix that now

**Review by olblak — APPROVED** [review]

**Pull request merged**